### DISCUSSION TITLE
Getting cors error while redirecting to payment gateway site

### DISCUSSION THREAD

**Discussion by mahmud081** [inertiajs/inertia#871] [Help]
I am getting below error while redirecting to payment gateway site:
![image](https://user-images.githubusercontent.com/8451947/130202722-b9f56021-eff4-473b-8387-dc4dfecc6476.png)

my payment gateway provider is using below code to redirect to their site:

header('Location: ' . $paymenturl . '/', true, $permanent ? 301 : 302);

Can any one help me on this issue??

**Comment by swiftalker** [comment]
try Inertia::location

**Reply by reinink** [reply]
Yup, this. See the "External redirects" section of the Inertia.js docs: https://inertiajs.com/redirects#external-redirects

**Comment by Michphkaymedia** [comment]
Use Inertia::location it's reload page.
